src/Microsoft.AspNet.Mvc.Xml/DefaultWrapperProviderFactoryProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class DefaultWrapperProviderFactoryProvider : IWrapperProviderFactoryProvider
```

Comment:
I like the property on the formatter much better. This enable doing work per formatter and per input/output